packages/contracts/contracts/minter-suite/Minters/MinterSetPriceV5.sol

```diff
@@ -0,0 +1,400 @@
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
nit: In general, we should order all functions in the minters according to: https://docs.soliditylang.org/en/latest/style-guide.html#order-of-functions

Additionally, similar to how you are already doing with the state variables, I think it would be good to denote sections of functions that are related to the different libraries, which might help when we plug in the different libraries to different minters.